### PR TITLE
Set return type on ember html safe function

### PR DESCRIPTION
**Pull request by BryanCrotaz**
typed-ember/ember-typings#77
typed-ember/ember-cli-typescript#232

### CODE REVIEW

File: types/ember/test/ember-tests.ts [hunk 1]
```diff
@@ -168,3 +168,6 @@ const component1 = Ember.Component.extend(mix1, mix2, {
     lyft: Ember.inject.service(),
     cars: Ember.computed.readOnly('lyft.cars'),
 });
+
+// make sure htmlSafe returns a SafeString
+Ember.String.htmlSafe("hello").toString();
```
Comment (mike-north) [inline types/ember/test/ember-tests.ts]: This doesn't necessarily assert that `htmlSafe` returns a string. you want something like
```ts
Ember.String.htmlSafe("hello") // $ExpectType string
```
see: https://github.com/Microsoft/dtslint#write-tests

**Review by mike-north — CHANGES_REQUESTED** [review]

**Comment by mike-north** [comment]
@RyanCavanaugh I think that @typescript-bot is misbehaving here. `Merge:YSYL` was added to a 3-hour-old issue

File: types/ember/test/ember-tests.ts (hunk 1 above)
Comment (BryanCrotaz) [reply types/ember/test/ember-tests.ts]: Thanks - didn't know about those asserts. Very nice.

**Review by mike-north — APPROVED** [review]
Wonderful. Thank you for the help @BryanCrotaz

**Pull request merged**